.travis.yml

```diff
@@ -8,11 +8,18 @@ python:
  - "3.4"
  - "3.5"
  - "pypy"
+ - "pypy3"
+ - "pypy3.3-5.2-alpha1"
  - "nightly"
-install: pip install .
-script: tox
-branches:
-  only:
-    - master
-matrix:
-  fast_finish: true
+
+before_install:
+ # Show the current setuptools version
+ - ORIG_SETUPTOOLS_VERSION=`python -c "import setuptools; print('%s' % setuptools.__version__)"`
+ - echo setuptools $ORIG_SETUPTOOLS_VERSION
+install:
+ - pip install wheel
+ - python setup.py install bdist_wheel
+ - pip install ./dist/tox_travis-*.whl
```

Comment:
OK, let me see if I got this straight: `python setup.py install bdist_wheel` installs the source package (approximately the same as `pip install .`) and simultanously _creates_ the wheel, and `pip install ./dist/tox-travis-*.whl` ensures that the wheel is installable. Is that correct? Is there any reason that we can't remove the `install` from that command, and only have the wheel being installed? It's the only thing that I really want to support.